### ISSUE TITLE
chmod u+x docker-entrypoint.sh && dependency

### ISSUE THREAD

**Issue by drPytho** [Netflix/dispatch#50]
```
root@917329a03ed4:/etc/dispatch# ./docker-entrypoint.sh server start
bash: ./docker-entrypoint.sh: Permission denied
root@917329a03ed4:/etc/dispatch# chmod u+x docker-entrypoint.sh
root@917329a03ed4:/etc/dispatch# ./docker-entrypoint.sh
root@917329a03ed4:/etc/dispatch# ./docker-entrypoint.sh server start
./docker-entrypoint.sh: line 17: exec: server: not found
root@917329a03ed4:/etc/dispatch# ./docker-entrypoint.sh "server start"
Traceback (most recent call last):
  File "/usr/local/bin/dispatch", line 5, in <module>
    from dispatch.cli import entrypoint
  File "/usr/local/lib/python3.8/site-packages/dispatch/cli.py", line 13, in <module>
    from dispatch.application.models import *  # noqa
  File "/usr/local/lib/python3.8/site-packages/dispatch/application/models.py", line 9, in <module>
    from dispatch.models import DispatchBase, applications_incidents, TimeStampMixin
  File "/usr/local/lib/python3.8/site-packages/dispatch/models.py", line 181, in <module>
    from dispatch.status_report.models import *  # noqa
ModuleNotFoundError: No module named 'dispatch.status_report'
root@917329a03ed4:/etc/dispatch#
```

After having built the image with `docker build . -t dispatch-local`

And running it with `docker run -ti --entrypoint=bash dispatch-local`

**Comment by drPytho** [comment]
Also just running it `docker run dispatch-local`
results in
`server: 1: exec: /etc/dispatch/docker-entrypoint.sh: Permission denied`

**Comment by drPytho** [comment]
I propose for the first issue to add under 
`COPY ./docker/docker-entrypoint.sh ./docker/.env $DISPATCH_CONF/`
`RUN chmod u+x $DISPATCH_CONF/docker-entrypoint.sh`

**Comment by drPytho** [comment]
The `ModuleNotFoundError: No module named 'dispatch.status_report'` I have no idea of how to fix.

**Comment by kevgliss** [comment]
`ModuleNotFoundError: No module named 'dispatch.status_report` should be fixed by #52

**Comment by drPytho** [comment]
Thanks, both issues are solved. Saw that @TheDahv made an improved solution too. :+1:
